### ISSUE TITLE
Install 7.1 and get core dump from mongod

### ISSUE THREAD

**Issue by shanedewitt** [projectcypress/cypress#1827]
I am using the VM install. After rebooting server, I try to access web page but it hangs.

I check running services and see that mongod is stopped and has a core dump.

```
× mongod.service - MongoDB Database Server
     Loaded: loaded (/lib/systemd/system/mongod.service; enabled; vendor preset: enabled)
     Active: failed (Result: core-dump) since Thu 2023-07-13 19:21:04 UTC; 48s ago
       Docs: https://docs.mongodb.org/manual
    Process: 741 ExecStart=/usr/bin/mongod --config /etc/mongod.conf (code=dumped, signal=ILL)
   Main PID: 741 (code=dumped, signal=ILL)
        CPU: 47ms

Jul 13 19:20:59 ubuntu systemd[1]: Started MongoDB Database Server.
Jul 13 19:21:04 ubuntu systemd[1]: mongod.service: Main process exited, code=dumped, status=4/ILL
Jul 13 19:21:04 ubuntu systemd[1]: mongod.service: Failed with result 'core-dump'.
```

**Comment by dczulada** [comment]
Shane-

When you are referring to 'VM install' are you referring to the download and installation here? https://github.com/projectcypress/cypress/wiki/Cypress-7-VM-Install-Instructions

Can you provide us a little more information on how you have it installed (e.g., what VM player) and we can try to recreate your issue.

-Dave Czulada

**Comment by shanedewitt** [comment]
Yes I use the image provided, but I convert it and load it into KVM.

`qemu-img convert Cypress.cypress*.vmdk Cypress.cypress-disk1.qcow2 -O qcow2`

My Host OS is Red Hat Enterprise Linux Server release 6.10 (Santiago).

I believe issue is mongo version 5 requires certain cpu flags and my virtualization host does not provide those.

I did successfully use your upgrade script to update my cypress instance.
